src/components/Kite/Kite.module.css

```diff
@@ -0,0 +1,88 @@
+.Kite {
+  z-index: var(--rvr-zindex-kite);
+  visibility: hidden;
+  box-sizing: border-box;
+  position: fixed;
+  top: 12px;
+  right: 12px;
+  transition: transform .6s ease-in-out;
+  animation: slide-in-right .7s;
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+  justify-content: center;
+  background-color: var(--rvr-white);
+  width: 342px;
+  box-shadow: 0px 5px 15px rgba(0, 0, 0, 0.2);
+  border-radius: 4px;
+  padding: 16px;
+  font-size: var(--rvr-font-size-md);
+  color: var(--rvr-color-font-dark);
+}
+
+.visible {
+  visibility: visible;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.content {
+  flex: 1 1 auto;
+  min-width: 0;
+  display: flex;
+  align-items: center;
+  width: 80%;
+}
+
+.childrenWrapper {
+  margin-top: 8px;
+  margin-bottom: 8px;
+}
+
+.dismissButton {
+  flex: 0 0 auto;
+  margin-right: -16px;
+}
+
+.icon {
+  display: flex;
+  flex: 0 0 auto;
+  margin-right: 16px;
+}
+
+.enterDone {
+  opacity: 1;
+  pointer-events: visible;
```

Comment:
I wasn't familiar with `visible` as an option for `pointer-events`, but it looks like it's for SVG only.
In this case, you probably don't even need it, since you're not setting `pointer-events` to a value other than `auto` in the first place.